rb_ws/src/buggy/buggy/simulator/engine.py

```diff
@@ -8,25 +9,20 @@
 from nav_msgs.msg import Odometry
 import numpy as np
 import utm
+sys.path.append("/rb_ws/src/buggy/buggy")
+from util import Constants
 
 class Simulator(Node):
-    # simulator constants:
-    UTM_EAST_ZERO = 589702.87
-    UTM_NORTH_ZERO = 4477172.947
-    UTM_ZONE_NUM = 17
-    UTM_ZONE_LETTER = "T"
-    #TODO: make these values accurate
-    WHEELBASE_SC = 1.17
-    WHEELBASE_NAND= 1.17
+
 
     def __init__(self):
         super().__init__('sim_single')
         self.get_logger().info('INITIALIZED.')
 
 
         self.starting_poses = {
```

Comment:
I feel these could also be going to constants, this feels kinda sketch ngl

Reply:
(by these are you referring to the start poses?) they are constants, but they are definitely not relevant to any system besides the sim so i left them in here to not clutter constants - easy fix if we want tho

Reply:
Let's keep it as is then.